franklin/builder/models.py

```diff
@@ -92,40 +89,62 @@ class Build(models.Model):
 
     :param site: Reference to the project for this build instance
     :param git_hash: If a branch build, the git hash of the deployed code
-    :param branch: If a branch build, the name of the branch
-    :param tag: If a tag build, the name of the tag
     :param created: Date this code was built
     :param path: The path of the site on the static server
     """
 
     site = models.ForeignKey(Site, related_name='builds')
-    git_hash = models.CharField(max_length=40, blank=True, null=True)
-    branch = models.CharField(max_length=100, blank=True, null=True)
-    tag = models.CharField(max_length=100, blank=True, null=True)
     created = models.DateTimeField(editable=False)
```

Comment:
Thinking on it more, I think we should use this new model inheritance code as it allows us to implement `unique`, `unique_together`, and `required` for the fields `git_hash`, `branch`, and `tag`. We couldn't do that with the previous model structure. Now we can enforce that. Implementing those now.